### ISSUE TITLE
IIS Examples

### ISSUE THREAD

**Issue by rhys-intelliflo** [PowerShellOrg/tug#89]
Is there any documentation or a walk through to configuring IIS to host tug specifically?

**Comment by FLeven** [comment]
Yes, I also feel a bit lost here. Some windows dockerfiles would be nice, this would ease the setup for a testrun.

**Comment by rhys-intelliflo** [comment]
+1 for a dockerfile

**Comment by affieuk** [comment]
This is how I got it working, would like to have to confirmation on this:

Install IIS
Install the .Net Core hosting bundle: https://aka.ms/dotnetcore-2-windowshosting
Install Tug
Create a site in IIS and point it to `C:\Program Files\Tug\Server`
Change application pool to `No Mananged Code`
Copy the web.config files from `C:\Program Files\Tug\Server\bin` to `C:\Program Files\Tug\Server`
Modify line 7 from `<aspNetCore processPath=".\Tug.Server.exe" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" />` to `<aspNetCore processPath=".\bin\Tug.Server.exe" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" />`
Test out Tug, haven't played with it further yet.

**Comment by rhys-intelliflo** [comment]
Thanks @affieuk will give this a go

**Comment by ebekker** [comment]
@affieuk directions reflect the current best practices for fronting ASP.NET Core app with IIS.  This is true up to .NET Core 2.0.

Using the instructions above will install a *bridging* component that proxies from the IIS process to the Kestrel process that's actually hosting and running the Tug server.

In the upcoming .NET Core 2.1 release, there will be a second option that allows for *in-process* support, so IIS process will actually spin up a .NET Core runtime in-process.  Early indications are that it will perform about 7x better.

**Comment by FLeven** [comment]
Install Tug -> The PS5 Version, as outlined int he pre-release deployment HOWTO ?
